src/main/kotlin/rmc/repository/user/UserRepositoryImpl.kt

```diff
@@ -53,4 +53,6 @@ class UserRepositoryImpl : UserRepository {
         if (!BCrypt.checkpw(user.password, userEntity.password)) throw AuthenticationFailed()
         return userEntity.toUserDto()
     }
-}
\ No newline at end of file
+}
+
+val userRepository: UserRepository = UserRepositoryImpl()
```

Comment:
Is deze nodig? En zo ja, waarom staat die niet direct na de imports?

Reply:
Ik instantieer hier een userRepository éénmalig in de UserRepository. In Kotlin kun je deze instantie importeren in andere files en gebruiker zonder dat je ze opnieuw hoeft te instantieren. 

Zo zie je dat ik in de RentalRoutes.kt op line 14 deze instantie importeer en op lijn 24 aanroep zonder dat hij geinstantieerd is in dit bestand.